eng/.docsettings.yml

```diff
@@ -16,6 +16,8 @@ required_readme_sections:
   - ^Contributing$
 known_presence_issues:
   - ['sdk/keyvault','#5499']
+  - ['sdk/eventhub','#5499']
 known_content_issues:
   - ['README.md','Root readme']
-  - ['sdk/keyvault/Microsoft.Azure.KeyVault/README.md','#5499']
\ No newline at end of file
+  - ['sdk/keyvault/Microsoft.Azure.KeyVault/README.md','#5499']
+  - ['sdk/eventhub/Microsoft.Azure.EventHubs/ReadMe.md','#5499']
```

Comment:
>eventhub [](start = 10, length = 8)

Service name is 'EventHubs' actually. We use 'event hub' to refer to single Event Hubs entity.

Reply:
Understood, and I've tried to honor that wherever possible.  That said, the repository structure is expected to snap to the names used in the [Azure REST API Specifications](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/eventhub/resource-manager) repository.   Unfortunately, in this case, they used `eventhub` which is why the root namespace lacks the proper `s`.